docs/content/manpages/geowave-addstore.txt

```diff
@@ -0,0 +1,31 @@
+//:geowave-addstore(1)
```

Comment:
Do you want to show arguments for when you specify -t accumulo?  There are additional arguments like username, password, etc...

"geowave config addstore -t accumulo" will show you additional arguments for accumulo.